introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi there, this is Oana.
 Favorite Code School path: JavaScript.
+*Git
+*JavaScript
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?